akka-docs-new/src/main/paradox/guide/modules.md

```diff
@@ -0,0 +1,165 @@
+# Akka Libraries and Modules
+
+Before we delve further into writing our first actors, we should stop for a moment and look at the set of libraries 
+that come out-of-the-box. This will help you identify which modules and libraries provide the functionality you 
+want to use in your system. 
+
+### Actors (`akka-actor` library, the core) 
+
+The use of actors across Akka libraries provides a consistent, integrated model that relieves you from individually 
+solving the challenges that arise in concurrent or distributed system design. From a birds-eye view, 
+actors are a programming paradigm that takes encapsulation (one of the pillars of OOP) to its extreme. 
+Unlike objects, actors encapsulate not only their 
+state but their execution. Communication with actors is not via method calls but by passing messages. While this 
+difference seems to be minor, this is actually what allows us to break clean from the limitations of OOP when it 
+comes to concurrency and remote communication. Don’t worry if this description feels too high level to fully grasp 
+yet, in the next chapter we will explain actors in detail. For now, the important point that this is a model that 
+handles concurrency and distribution at the fundamental level instead of ad hoc patched attempts to bring these 
+features to OOP.
+
+Problems that actors solve include:
+
+* How to build and design high-performance, concurrent applications?
+* How to handle errors in a multi-threaded environment?
+* How to protect my project from the pitfalls of concurrency?
+
+### Remoting
+
+Remoting enables actors that are remote from each other, living on different computers, to seamlessly exchange messages. 
+While distributed as a JAR artifact, Remoting resembles a module more than it does a library. You enable it mostly 
+with configuration, it has only a few APIs. Thanks to the actor model, remote and local message sends look exactly the 
+same. The patterns that you use locally on multi-core systems translate directly to remote systems. 
+You will rarely need to use Remoting directly, but it provides the foundation on which the Cluster subsystem is built.
+ 
+Some of the problems Remoting solves are
+
+* How to address actor systems living on remote hosts?
+* How to address individual actors on remote actor systems?
+* How to turn messages to bytes on the wire?
+* How to manage low-level, network connections (and reconnections) between hosts, detect crashed actor systems and hosts, 
+  all transparently?
+* How to multiplex communications from unrelated set of actors on the same network connection, all transparently?
+
+### Cluster
```

Comment:
mention (`akka-cluster` library) if we want to be consistent with akka-actor in this section?